src/pal/src/misc/miscpalapi.cpp

```diff
@@ -454,3 +454,159 @@ EnsureOpenSslInitialized()
     return dwRet;
 }
 
+/*++
```

Comment:
The functions are there, but C++ mangled.

```
0000000000000200 T _Z24EnsureOpenSslInitializedv
```

Your test is a .c file tho, so its probably not mangling?